css-will-change-transform-rasterization/demo.js

```diff
@@ -0,0 +1,9 @@
+window.addEventListener('load', function() {
```

Comment:
It's required or else the timing is such that the transformed text won't be blurry. (The logic kicks in for transformations that happen after the page is loaded?)